ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js

```diff
@@ -142,6 +142,8 @@ export function CatalogItemModalController($scope, $filter, blueprintService, pa
         saving: false,
         force: false,
     };
+    $scope.catalogBundleId ="";
+    $scope.catalogBundleBase ="";
```

Comment:
`""` will generate incorrect URL so not sure it's worth initializing with invalid value, but it should get populated quickly so NBD